spec/support/generator_test_helpers.rb

```diff
@@ -5,14 +5,20 @@ module GeneratorTestHelpers
   include TestPaths
   include FileOperations
 
-  def invoke!(klass, *args, **kwargs)
+  def invoke!(klass, *args, **kwargs, &block)
```

Comment:
Changes in this file overlap with the same changes [in this PR](https://github.com/thoughtbot/suspenders/pull/1076). It will be fixed during merge.